Dockerfile

```diff
@@ -1,12 +1,15 @@
 FROM jruby:9.2.5.0-jdk-alpine
-RUN apk add --no-cache --update build-base postgresql-dev nodejs curl
-RUN mkdir /app
+
 WORKDIR /app
+
 COPY lib lib
-RUN ./lib/upvs/compile
-RUN gem install bundler
-RUN gem update --system
 COPY Gemfile Gemfile.lock ./
-RUN bundle install --without development:test --path vendor/bundle --deployment
```

Comment:
Tuto by sa mozno dali riadky 7 a 8 spojit, ale skor treba zistit tam preco tam vobec je `install bundler` a `update`. Predpokladam ze to bola nejaka broken dependency v jruby.